sphinx_gallery/gen_rst.py

```diff
@@ -1237,21 +1268,32 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf, seen_backrefs=No
         Absolute path to directory where source examples are stored.
     gallery_conf : dict
         Contains the configuration of Sphinx-Gallery.
-    seen_backrefs : set
-        The seen backreferences.
 
     Returns
     -------
     intro: str
         The introduction of the example.
+    title : str
+        The example title.
     cost : tuple
         A tuple containing the ``(time_elapsed, memory_used)`` required to run the
         script.
+    out_vars : dict
+        Variables used to run the script, possibly with entries:
+
+        "stale"
+            True if the example was stale.
```

Comment:
This doc looks incorrect? On line 1315, the 'stale' key is set to a string of something.